ui/vendor/css/bootstrap.css

```diff
@@ -2423,12 +2633,12 @@ input[type="month"].input-lg {
 .checkbox {
   position: relative;
   display: block;
-  min-height: 20px;
```

Comment:
Fwiw, this removal of `min-height` means we've lost our min height on our Similar Jobs checkboxes per screen grab below. They are up against the bottom of the grey container in stead of having 2px of breathing space below them.

![checkboxminheight](https://cloud.githubusercontent.com/assets/3660661/8854435/e021e3b6-312d-11e5-8130-204bd41f72a8.jpg)